proposed/swift-interop.md

```diff
@@ -152,33 +152,46 @@ If possible, Swift tuples should be represented as `ValueTuple`s in .NET. If thi
 
 #### Projection Tooling Components
 
-The projection tooling should be split into these components:
+The projection tooling will be based on the [Binding Tools for Swift](https://github.com/xamarin/binding-tools-for-swift) (BTfS). The BTfS contains a set of tools that can consume a compiled Apple Swift library and generate wrappers (bindings) that allow it to be surfaced as a .NET library. This tool will be implemented as a self-hosted .NET CLI tool, automating the generation of wrappers by parsing the Swift library interface. It generates C# bindings that can be utilized as a .NET library and Swift bindings that C# bindings invoke in cases where direct P/Invoke into Swift library is not possible.
+
+The projection tooling should be split into these components.
 
 ##### Importing Swift into .NET
```

Comment:
A more appropriate design for this repo and document might be a simple flow sequence. For example, for COM I would typically describe the design of the importing COM into .NET as follows.

```md
### Import COM into .NET

Importing COM into .NET is done through the following steps.

1) The COM definition is contained in a `.idl` or `.tlb` (Type Library) file.

2) If the file is a `.tlb` goto step (3). For an `.idl`, the file is passed to the MIDL compiler, which converts it to a `.tlb` file.

3) The `.tlb` is passed to the `TlbImp` tool, which will convert the TypeLibrary into a .NET Assembly.
```

For Swift there might be more details, but in general the design of the import should speak of user artifacts that are obvious. An example of them might be `.swift` files, `.dylib` files, `.dySYM` directory, header files, .NET Assemblies, other artifacts that may control configuration or provide information needed for the conversion.

Reply:
I've updated the flow to avoid disclosing tool-specific details and highlights artifacts.